src/index.html

```diff
@@ -11,7 +11,134 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend class="leg">Personal information</legend>
+        <label class="form">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+
+        </label>
+        <label class="form">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+        <label class="form">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form">
+          Password:
+          <input
+            type="password"
+            name="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="love cats"
+            > Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="love cats"
+            > No
+          </label>
+        </div>
+        <label class="form">
+          What is your favorite color?
+          <input
+            type="color"
+            value="black"
+          >
+        </label>
+
+        <label class="form"> What time do you go to bed?
+          <input
+            type="time"
+          >
+        </label>
```

Comment:
```suggestion
        <label class="form">
          What time do you go to bed?
          <input type="time">
        </label>
```
write tags in 1 line if we have less than 3 attributes